wagtail/images/templatetags/wagtailimages_tags.py

```diff
@@ -107,3 +110,14 @@ def render(self, context):
             for key in self.attrs:
                 resolved_attrs[key] = self.attrs[key].resolve(context)
             return rendition.img_tag(resolved_attrs)
+
+
+@register.simple_tag()
+def image_url(image, filter_spec, viewname='wagtailimages_serve'):
+    try:
+        return generate_image_url(image, filter_spec, viewname)
+    except NoReverseMatch:
+        raise template.TemplateSyntaxError(
```

Comment:
Hm. I'm not sure if `TemplateSyntaxError` is a good option. Syntax of the template is correct, but the configuration is not.

Maybe we should use Django's `ImproperlyConfigured`? We use it widely in Wagtail.

Other than this, looks good